### PR TITLE
Extracted html from transformation_overview.php to template/transformation_overview.twig

### PR DESCRIPTION
**Pull request by sharik709**
Signed-off-by: Sharik Shaikh <shaikhsharik709@gmail.com>

### Description

Extracted html from transformation_overview.php to template/transformation_overview.twig

Fixes #

Before submitting pull request, please review the following checklist:

- [Yes] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [Yes] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [Yes] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [Yes] Every commit has a descriptive commit message.
- [Yes] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [Yes] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Hi, @sharik709. Thanks for your contribution.

File: templates/transformation_overview.twig [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<h2>{{ label.available_MIME_types }}</h2>
+
+{% for mimeType in types.mimetype %}
+	{% if types.empty_mimetype.mimetype %}
+		<i>{{ mimeType | escape }}</i>
```
[the remark this answers is not in the document]
Comment (sharik709) [reply templates/transformation_overview.twig]: OK got it. Ill remove them.

File: templates/transformation_overview.twig [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<h2>{{ label.available_MIME_types }}</h2>
```
[the remark this answers is not in the document]
Comment (sharik709) [reply templates/transformation_overview.twig]: Ok. Ill change it.

File: templates/transformation_overview.twig [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{% for ttype in transformation_types %}
+	<a name="{{ ttype.title | escape }}"></a>
+    <h2>{{ ttype.label | escape }}</h2>
+	<table width="90%">
+		<thead>
+			<th>
+				{{ table_headings_label[ttype.title] | escape }}
+			</th>
+			<th>
+				{{ ttype._pgettext | escape }}
```
[the remark this answers is not in the document]
Comment (sharik709) [reply templates/transformation_overview.twig]: ok. Ill replace it.

File: templates/transformation_overview.twig [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<h2>{{ label.available_MIME_types }}</h2>
+
+{% for mimeType in types.mimetype %}
```
[the remark this answers is not in the document]
Comment (sharik709) [reply templates/transformation_overview.twig]: ok.

File: templates/transformation_overview.twig [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	{% if types.empty_mimetype.mimetype %}
+		<i>{{ mimeType | escape }}</i>
+	{% else %}
+		{{ mimeType | escape }} <br>
+	{% endif %}
+{% endfor %}
+<br />
```
[the remark this answers is not in the document]
Comment (sharik709) [reply templates/transformation_overview.twig]: Ok got it.

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**